src/plugins/tree/treeBodyChildren.tmpl.html

```diff
@@ -7,7 +7,7 @@
             <a ng-disabled="!childrenRows || childrenRows.length === 0" data-nodrag
```

Comment:
I think it would be better to move this expression in `tree.controller.js` into an new `$scope.isCollapseDisabled()` function. And call this function there.